qiskit/transpiler/passes/calibration/pulse_gate.py

```diff
@@ -95,4 +95,4 @@ def get_calibration(self, node_op: CircuitInst, qubits: List) -> Union[Schedule,
         Raises:
             TranspilerError: When node is parameterized and calibration is raw schedule object.
         """
-        return self.target.get_calibration(node_op.name, tuple(qubits), *node_op.params)
+        return self.target.get_calibration(node_op.name, tuple(qubits), None, *node_op.params)
```

Comment:
Reviewer's comment anticipated! :) 
I am not super satisfied with putting `None` here.
Here, the parameter `node_op.params` has the actual list of float parameters to bind with the Gate's parameters stored in the target which is a parameter(variable). That is why I had to put `None` just to keep the function's behavior similar to its previous version.